test/e2e/run-test.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
 
-while ! curl -f -s -o /dev/null "http://backoffice:3000"
+adminConsoleHost="localhost"
+adminConsolePort="3000"
+if [[ -z "$e2eLocal" ]]; then
+  adminConsoleHost="adminconsole"
+fi
+
+while ! curl -f -s -o /dev/null "http://$adminConsoleHost:$adminConsolePort"
 do
-    echo "[$(date --rfc-3339 seconds)] - Waiting for http://backoffice:3000"
+    echo "[$(date --rfc-3339 seconds)] - Waiting for Kuzzle Admin Connsole to be up at http://$adminConsoleHost:$adminConsolePort"
     sleep 5
 done
 
 set -e
 
-testim --token $TESTIM_TOKEN --project $TESTIM_PROJECT --host "hub" --browser $BROWSER --label firstAdmin --base-url http://backoffice:3000 --report-file testim-report.xml --config-file /opt/config-file.js
-testim --token $TESTIM_TOKEN --project $TESTIM_PROJECT --host "hub" --browser $BROWSER --label normal --base-url http://backoffice:3000 --report-file testim-report.xml --config-file /opt/config-file.js
-testim --token $TESTIM_TOKEN --project $TESTIM_PROJECT --host "hub" --browser $BROWSER --label securityUser --base-url http://backoffice:3000 --report-file testim-report.xml --config-file /opt/config-file.js
+npm i
```

Comment:
I don't know if this will be the case, but can that script be started from inside a docker container?
If so, then you should add the `--unsafe-perm` argument

Reply:
It does run in a container, so I think it's a good idea to unsafe-perm it